enterprise/views.py

```diff
@@ -774,6 +782,9 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             organization_logo = None
             organization_name = None
 
+        if enterprise_course_enrollment and not enterprise_course_enrollment.consent_granted:
+            add_consent_declined_message(request, enterprise_customer, course_details)
```

Comment:
Since the `UserDataSharingConsentAudit` model is going to be short-lived and we are not actually going to make use of account-level consent in the short term, I'm thinking we should only consider course-level consent here. Let me know if there is a strong argument to do otherwise.